hack/ec-checks.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -e -o pipefail
+
+# extract the build container task name
+function build_container_name {
```

Comment:
Nitpickery, perhaps?

```suggestion
function build_container_task_name {
```